### ISSUE TITLE
New Option '--skip-null' causes ValueError raised by argparse.py

### ISSUE THREAD

**Issue by stefan-rademacher-tk** [github/saml-ldap-team-sync#19]
Hi,

since these lines (https://github.com/github/saml-ldap-team-sync/commit/7e11e7c1d1e31af6f7bdc876891a4da2daa59a57#diff-8cf96b3efbca5b77c8394b54670de4ddR93-R94) were added, I get an error, running the script:

```
Traceback (most recent call last):
  File "./SAMLTeamSyncAD.py", line 197, in <module>
    main()
  File "./SAMLTeamSyncAD.py", line 94, in main
    help="Skip empty groups in Active Directory, to avoid emptying the GitHub group")
  File "/usr/lib/python3.7/argparse.py", line 1362, in add_argument
    action = action_class(**kwargs)
  File "/usr/lib/python3.7/argparse.py", line 869, in __init__
    raise ValueError('nargs must be %r to supply const' % OPTIONAL)
ValueError: nargs must be '?' to supply const
```
I have these python modules installed:
```
Package           Version   
----------------- ----------
certifi           2019.11.28
cffi              1.14.0    
chardet           3.0.4     
cryptography      2.8       
defusedxml        0.6.0     
Deprecated        1.2.7     
idna              2.9       
jira              2.0.0     
ldap3             2.7       
oauthlib          3.1.0     
pbr               5.4.4     
pip               20.0.2    
pyasn1            0.4.8     
pycparser         2.20      
PyGithub          1.47      
PyJWT             1.7.1     
PyYAML            5.3.1     
requests          2.23.0    
requests-oauthlib 1.3.0     
requests-toolbelt 0.9.1     
setuptools        46.0.0    
six               1.14.0    
urllib3           1.25.8    
wrapt             1.12.1    
```

The error occurs, no matter if I run the script with or without --skip-null option.
I didn't take the time yet, to dive deeper. Does anyone know, why this is happening?

**Comment by timhirsh** [comment]
I'm seeing the same thing.  It looks like this was introduced in https://github.com/github/saml-ldap-team-sync/pull/17, which was a partial fix for https://github.com/github/saml-ldap-team-sync/issues/6.

**Comment by primetheus** [comment]
@stefan-rademacher-tk @timhirsh this is fixed in #24. Thanks!
